inc/functions/formatting.php

```diff
@@ -433,6 +391,13 @@ function rocket_get_cache_busting_paths( $filename, $extension ) {
  * @return string Resolved file path
  */
 function rocket_realpath( $file ) {
+	$wrapper = null;
+
+	// Strip the protocol.
+	if ( wp_is_stream( $file ) ) {
```

Comment:
Use `rocket_is_stream` instead. It's available in `develop` branch.

Reply:
Done https://github.com/wp-media/wp-rocket/commit/4f2bfb5985ec5d3363c3e1707b57a40ae1603a4e